<launch>
    <node pkg="hsi_driver" type="cube_data.py" name="cube_processor">
        <param name="camera_model" value="ximea"/>
    </node>
    <node pkg="hsi_driver" type="hsi_driver.py" name="hsi_processor">
        <param name="camera_model" value="ximea"/>
    </node>
    <node pkg="hsi_driver" type="combined_cube_data.py" name="combined_cube_processor">
        <param name="camera_model" value="ximea"/>
    </node>
</launch>
